### ISSUE TITLE
GOTO Addon - using WGS84 number of decimals insufficient

### ISSUE THREAD

**Issue by catmorales** [georchestra/georchestra#1885]
In WGS 84 , 2 decimals is not enough.
![image](https://user-images.githubusercontent.com/4659829/35523212-a0c25e5e-051e-11e8-809c-da640d2b74ba.png)

It could be interesting to have a parameter "decimals" in config.json file like coordinates addon.

`"decimals": 10,`

**Comment by fvanderbiest** [comment]
Right, this could be implemented in https://github.com/georchestra/georchestra/blob/16.12/mapfishapp/src/main/webapp/app/addons/goto/js/main.js#L116-L138 with http://docs.sencha.com/extjs/3.4.0/source/NumberField.html#Ext-form-NumberField-cfg-decimalPrecision

**Comment by severo** [comment]
Note that:
- the decimal separator depends on the language ("." in Spanish). I think it has no importance yet,
- if typing more than 2 decimal, the value is rounded to 2 decimal on Tab key event, but not on Enter key event -> is it a bug?
